privacyidea/lib/security/default.py

```diff
@@ -43,12 +43,11 @@
 import os
 
 from Crypto.Cipher import AES
-from privacyidea.lib.crypto import zerome
+from privacyidea.lib.crypto import zerome, _to_bytes, _to_unicode, hexlify_and_unicode
```

Comment:
if we use the functions `_to_bytes` and `_to_unicode` outside of the original module I think we should calls them `to_bytes` and `to_unicode`.

Reply:
I tried to keep these functions to the crypto.py module... I'll check if the import can be avoided.

Reply:
Maybe they'll be useful for porting other modules. At least ``hexlify_and_unicode`` might be :-)

Reply:
One Problem is, if we define something in `lib/utils.py` we can't use it in `lib/crypto.py` since the `crypto` module is imported into `utils`...
I would prefer if the `utils` module would not import any privacyIDEA specific modules so that we can import it everywhere.
We could  move the functions that make use of the `crypto` module (specifically `urandom()` and `geturandom()`) into the `crypto` module, but i would like to do that in a different PR.

Reply:
Yes, a privacyIDEA-independent utils module would be nice! But doing this in another PR is probably a good idea.